support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -742,6 +742,29 @@ form {
   margin-left: 5px;
 }
 
+.has-existing-payment-option-explainer {
```

Comment:
the radios in `/components/forms/customFields/radioInput.jsx`support images now so perhaps this'd be a good time to migrate to them? Might be a tall order though

Reply:
good shout - although I'd prefer to refactor the new and existing payment methods all at the same time, I'll make a card to do a follow up PR